app/timetables/tests/test_models.py

```diff
@@ -235,3 +235,20 @@ def test_zero_cycle_day_value_cannot_be_saved(self):
             timetable=self.timetable_object)
 
         self.assertRaises(ValidationError, menu_item_three.save)
+
+
+class VendorTest(TestCase):
+    """Test the Vendor model."""
+
+    def setUp(self):
+        Vendor.objects.create(
+            name='Spicy Foods',
+            info='Reach us at info@spicy-foods.com',
```

Comment:
In this test, the slug field is not required for creating a Vendor. Would that mean that the slug is used for some internal logic? In that case, should that field be added to the admin, and made editable by the admin?